lib/web_ui/dev/test_runner.dart

```diff
@@ -619,6 +619,8 @@ class TestCommand extends Command<bool> with ArgUtils {
       '--enable-experiment=non-nullable',
       '--no-sound-null-safety',
       if (input.forCanvasKit) '-DFLUTTER_WEB_USE_SKIA=true',
+      if (!input.forCanvasKit) '-DFLUTTER_WEB_AUTO_DETECT=false',
+      if (!input.forCanvasKit) '-DFLUTTER_WEB_USE_SKIA=false',
```

Comment:
nit: just so it doesn't look like there's a possibility of having `FLUTTER_WEB_USE_SKIA` twice we could do:

```
'-DFLUTTER_WEB_USE_SKIA=${input.forCanvasKit}',
```